Makefile

```diff
@@ -72,9 +72,14 @@ coverage:
 update-devel-image: TAG=devel
 update-devel-image: TMPDIR:=$(shell mktemp -d)
 update-devel-image:
-	@CGO_ENABLED=0 GOOS=linux go build -o $(TMPDIR)/forwarder ./cmd/forwarder
 	@ln Dockerfile LICENSE LICENSE.3RD_PARTY $(TMPDIR)
-	@docker buildx build -t saucelabs/forwarder:$(TAG) $(TMPDIR)
+ifeq ($(shell uname),Linux)
+	@CGO_ENABLED=1 GOOS=linux go build -race -o $(TMPDIR)/forwarder ./cmd/forwarder
+	@docker buildx build --build-arg BASE_IMAGE=ubuntu:latest -t saucelabs/forwarder:$(TAG) $(TMPDIR)
```

Comment:
Why base image is different?

Reply:
distroless:static does not have libc

Reply:
Gotcha how about `base`

https://github.com/GoogleContainerTools/distroless/blob/main/base/README.md

Reply:
Base image also had some issues. 
You can check the pipeline with base image here: https://github.com/saucelabs/forwarder/actions/runs/6932307079/job/18855937399